CHANGES.rst

```diff
@@ -91,6 +91,10 @@ astropy.table
 - ``MaskedColumn.data`` will now return a plain ``MaskedArray`` rather than
   the previous (unintended) ``masked_BaseColumn``. [#8855]
 
+- Added a sentence in documentation to mention the warning that appears with the new
```

Comment:
I am actually not sure if a change log is necessary. @bsipocz ?

Reply:
I can remove that if you want me to.. I was not exactly sure what are the rules on that.

Reply:
Usually doc changes don't need a change log but there are exceptions. Let's wait to see what Brigitta says.

Reply:
yes, please remove the changelog entry.

Rule of thumb is to list changes that are related to the API the users are coming in contact with.